WooCommerce/Classes/ViewRelated/Orders/Order Creation/ProductsSection/ConfigurableBundleItemView.swift

```diff
@@ -30,38 +31,59 @@ struct ConfigurableBundleItemView: View {
                     .errorStyle()
             }
 
-            Button {
-                viewModel.createVariationSelectorViewModel()
-                showsVariationSelector = true
-            } label: {
-                Text(viewModel.selectedVariation == nil ?
-                     Localization.selectVariation: Localization.updateVariation)
-            }
-            .renderedIf(viewModel.isVariable)
-
-            if let selectedVariation = viewModel.selectedVariation {
+            HStack(spacing: 0) {
+                // Indentation from the checkmark image.
                 Spacer()
-                    .frame(height: Layout.defaultPadding)
+                    .frame(width: Layout.checkmarkDimension + Layout.defaultPadding)
```

Comment:
I have run into something like this before, and for now this seems like the best solution given that this constant can be used for both the checkmark and the spacer to keep their dimensions the same.

In the future, it looks like [Grid](https://developer.apple.com/documentation/swiftui/grid) could be a good option for layouts like this (iOS 16+).

Reply:
> In the future, it looks like [Grid](https://developer.apple.com/documentation/swiftui/grid) could be a good option for layouts like this (iOS 16+).

Nice TIL! The iOS 16+ Grid looks nice 👍